spa/src/components/donations/filters/StatusFilter.js

```diff
@@ -7,22 +7,25 @@ import { PAYMENT_STATUS } from 'constants';
 
 const STATUS_FILTERS = Object.values(PAYMENT_STATUS);
 
-function StatusFilter({ filter = [], handleFilterChange }) {
+function StatusFilter({ filter = [], handleFilterChange, excludeStatusFilters = [] }) {
   return (
     <FilterWrapper data-testid="status-filter">
       <S.StatusFilter>
         <FilterLabel>Status:</FilterLabel>
         <S.Statuses>
-          {STATUS_FILTERS.map((f) => (
-            <S.StatusBadge
-              key={f}
-              data-testid={`status-filter-${f}`}
-              selected={filter.includes(f)}
-              onClick={() => handleFilterChange('status', f)}
-            >
-              <StatusCellIcon status={f} showText size="sm" />
-            </S.StatusBadge>
-          ))}
+          {STATUS_FILTERS.map(
+            (f) =>
+              excludeStatusFilters.indexOf(f) == -1 && (
+                <S.StatusBadge
+                  key={f}
+                  data-testid={`status-filter-${f}`}
+                  selected={filter.includes(f)}
+                  onClick={() => handleFilterChange('status', f)}
+                >
+                  <StatusCellIcon status={f} showText size="sm" />
+                </S.StatusBadge>
```

Comment:
Also, I didn't see this when I edited this page, but could you rename `f` for `status` or another more descriptive name? 
`f` is really hard to understand what it stands for 
```suggestion
                  key={status}
                  data-testid={`status-filter-${status}`}
                  selected={filter.includes(status)}
                  onClick={() => handleFilterChange('status', status)}
                >
                  <StatusCellIcon status={status} showText size="sm" />
                </S.StatusBadge>
```